app/commands/track/set_file_system_permissions.rb

```diff
@@ -0,0 +1,27 @@
+class Track::SetFileSystemPermissions
+  include Mandate
+
+  initialize_with :track
+
+  def call
+    return if safe_directory_already_set?
+
+    add_safe_directory!
+
+    # A redeploy of the website is required to pick up the new safe directory
+    trigger_redeploy!
+  end
+
+  private
+  def safe_directory_already_set?
+    Kernel.system("git config --global --get safe.directory #{safe_directory}")
```

Comment:
According to the docs, `system` will return `true` when the exit code is 0, and `false` when it isn't zero. And conveniently, the git command returns 1 when the safe directory isn't set, and 0 when it is